### ISSUE TITLE
specified logon session does not exist

### ISSUE THREAD

**Issue by ChipGandy** [dfinke/ImportExcel#952]
I'm running powershell as another user for this script and was getting an error when running export-excel.

PS C:\Temp> Export-Excel -InputObject $d -Now
Invoke-Item : A specified logon session does not exist. It may already have been terminated
At C:\Users\p1cgandy\Documents\WindowsPowerShell\Modules\ImportExcel\7.1.1\Public\Export-Excel.ps1:694 char:26
+             if ($Show) { Invoke-Item $Path }
+                          ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Invoke-Item], Win32Exception
    + FullyQualifiedErrorId : System.ComponentModel.Win32Exception,Microsoft.PowerShell.Commands.InvokeItemCommand

in this case, inputobject was just get-service

**Comment by dfinke** [comment]
Does it make a difference if you drop the `-Now` ?

**Comment by ChipGandy** [comment]
Negative.

I saw some articles regarding running excel under RunAs.  I went to another system and logged in directly with that account and the account worked.  Working backwards I did runas with my 1st account and was able to export.  I flipped around and was able to get it to work as well.

Excel patched 13001.20266 no longer allowing runas - Microsoft Community<https://answers.microsoft.com/en-us/msoffice/forum/all/excel-patched-1300120266-no-longer-allowing-runas/4f0002e3-6d43-4c94-ae36-47e00feb774e>

I wouldn’t think this would matter since excel isn’t being invoked correct?

I need to try from my laptop the same thing of logging in directly first.  I normally only do runas but in the end, this will be running as a task under credentials anyway.

Thanks!

COX
----------------------------------------------------------------------------
Chip Gandy
Enterprise Software Specialist IV, Desktop Engineering
404-269-6039 tel
6305-A Peachtree Dunwoody Road, Atlanta GA 30328
